compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -140,11 +120,7 @@ internal class ComposeContainer(
             onWindowContainerPositionChanged()
         }
 
-    val architectureComponentsOwner = object : PlatformArchitectureComponentsOwner {
-        override val lifecycleOwner get() = this@ComposeContainer
-        override val navigationEventDispatcherOwner get() = this@ComposeContainer
-        override val viewModelStoreOwner get() = this@ComposeContainer
-    }
+    val architectureComponentsOwner = DefaultArchitectureComponentsOwner(savedState)
```

Comment:
It seems it's public only for tests. If so, it should be
```suggestion
    @VisibleForTesting
    val architectureComponentsOwner = DefaultArchitectureComponentsOwner(savedState)
```